tests/factories/cht/generate.js

```diff
@@ -87,23 +119,38 @@ const createAdditionalPersons = (nbrPersons, clinic) => {
   return Array
     .from({ length: nbrPersons - 1 })
     .map((_, i) => {
-      const additionalPersonName = `${FIRST_NAMES[i % FIRST_NAMES.length]} ${LAST_NAMES[i % LAST_NAMES.length]}`;
+      const additionalPersonName = `${
+        ADDITIONAL_PERSON_FIRST_NAMES[i % ADDITIONAL_PERSON_FIRST_NAMES.length]
+      } ${clinic.last_name}`;
```

Comment:
I think this is better than splitting the string
```suggestion
       const name = ADDITIONAL_PERSON_FIRST_NAMES[i % ADDITIONAL_PERSON_FIRST_NAMES.length];
       const additionalPersonName = `${name} ${clinic.last_name}`;
```